README.md

```diff
@@ -107,6 +107,30 @@ $ ./mvnw -DskipTests --also-make -pl zipkin-server clean install
 $ java -jar ./zipkin-server/target/zipkin-server-*exec.jar
 ```
 
+## Zipkin Filters
+
+Zipkin supports a plugin model for filters. You can take filters off the shelf or create your own. There are two classes
+of filters: a) Span Filters - where you want to operate on individual spans. b) Http Filters - where you want to operate
+on headers. If you want to create your own, please follow the examples mentioned into the links below.
+
+### Zipkin Span Filters
+
+Here is an example enrichment filter that allows you to inject a tag to your spans from environment passed to Zipkin:
+[zipkin-generic-enrichment-filter](https://github.com/afalko/zipkin-generic-enrichment-filter)
+
+### Zipkin Http Filters
```

Comment:
please remove forward refs to http filters which I am not sure what they will be. makes the change easier to pin down.